src/main/groovy/wooga/gradle/build/unity/UnityBuildPlugin.groovy

```diff
@@ -100,7 +100,9 @@ class UnityBuildPlugin implements Plugin<Project> {
                     def excludeSpec = new Spec<FileTreeElement>() {
                         @Override
                         boolean isSatisfiedBy(FileTreeElement element) {
-                            return extension.ignoreFilesForExportUpToDateCheck.contains(element.getFile())
+                            return extension.ignoreFilesForExportUpToDateCheck.any {
+                                f -> element.getFile().name.contains(f.name)
```

Comment:
Help me out here. What is the expectation of this change?
I can't wrap my head around this change at the moment :)

I think you actually want to compare the path not only the file names or? That could lead to issues I think.

Reply:
I think you should first check if `f` is a directory and then decide to to a check if `element.getFile().getPath()` starts with this path. If it is a file, to a normal comparison.